lib/flop_phoenix/table.ex

```diff
@@ -143,6 +144,41 @@ defmodule Flop.Phoenix.Table do
     """
   end
 
+  attr :field, :atom, required: true
+  attr :label, :string, required: true
+  attr :path, :string
+  attr :on_sort, JS
+  attr :event, :string
+  attr :target, :string
+
+  def sort_link(%{event: event} = assigns) when is_binary(event) do
```

Comment:
Exposed this as public in order to allow consumers to compose this with their implementation. Taking a step back, it might be worth pulling out the direct usage of label so consumers can wrap any element they want in a link.